draft-ietf-rtcweb-jsep.xml

```diff
@@ -1984,7 +1984,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               be 3 bytes or less, to allow them to efficiently fit into
```

Comment:
Might want to just write this as a single sentence, e.g.
"When generating mid values, they MUST be generated in a fashion that does not leak user information, e.g. randomly or from a counter, and SHOULD be 3 bytes of less, to allow..."